product_docs/docs/pgd/5/security/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Security and roles
+redirects: 
+  - ../bdr/security
+navigation:
+- roles
+- pgd-predefined-roles
+- roles-and-replication
+- access-control
+- role-management
+- verification
+---
+
+EDB Postgres Distributed allows a PGD cluster to be administered without giving access to the stored data by design. It achieves this through the use of roles and controlled access to system objects.
+
+* [Roles](roles) introduces the roles that PGD predefines for controlling access to PGD functionality.
+
+* [PGD predefined roles](pgd-predefined-roles) details the specific privileges of the PGD roles.
+
+* [Roles and replication](roles-and-replication) explains how PGD replication interacts with roles and privileges.
+
+* [Access control](access-control) explains how tables, functions, catalog objects and triggers interact with PGD roles and Postgres attributes.
+
+* [Role management](role-management) discusses how roles are managed on multi-database nodes and new nodes.
+
```

Comment:
I suggest moving this section above access control, to have all role-related topics grouped together in the index.